File: src/main/java/jp/kentan/minecraft/neko_event/listener/SignListener.java
package jp.kentan.minecraft.neko_event.listener;

import org.bukkit.block.Sign;
import org.bukkit.event.block.SignChangeEvent;
import org.bukkit.event.player.PlayerInteractEvent;

public interface SignListener {
    void onSignChanged(SignChangeEvent event);
    void onPlayerInteract(PlayerInteractEvent event, Sign sign);
}

File: src/main/java/jp/kentan/minecraft/nekoevent/listener/SignListener.kt
package jp.kentan.minecraft.nekoevent.listener

import org.bukkit.block.Sign
import org.bukkit.event.block.SignChangeEvent
import org.bukkit.event.player.PlayerInteractEvent

interface SignListener {
    fun onSignChanged(event: SignChangeEvent)
    fun onPlayerInteract(event: PlayerInteractEvent, sign: Sign)
}